### ISSUE TITLE
Everytime Update whitelist on DockerHub Remote repository, password gets empty

### ISSUE THREAD

**Issue by EvertonSA** [jfrog/terraform-provider-artifactory#642]
I as a platform engineer, when I'm adding a new whitelisted image to dockerhub remote, the docker password always gets overwritten with empty value.

before apply:
dockerhub credentials are correctly set.

after apply:
dockerhub username is there but password is empty. if no manual intervention is done, remote gets unavailable (due to reaching dockerhub pull limit). 

```
resource "artifactory_remote_docker_repository" "registry_1_docker_io" {
  key                           = "registry-1.docker.io"
  url                           = "https://registry-1.docker.io/"
  proxy                         = "Proxy"
  includes_pattern              = <<EOT
    alpine/git/**,
    alpine/helm/**,
   ...
    zricethezav/gitleaks/**
  EOT
  external_dependencies_enabled = false
  block_pushing_schema1         = false
  xray_index                    = false
  property_sets = [
    "artifactory",
  ]
  lifecycle {
    ignore_changes = [
      username,
      password
    ]
  }
}
```

provider version:
- Installing jfrog/artifactory v6.15.1...
- Installed jfrog/artifactory v6.15.1 (unauthenticated)

Is anything wrong with my config?

**Comment by danielmkn** [comment]
Please, check with the latest version v6.26.0

**Comment by danielmkn** [comment]
@EvertonSA thanks for the ticket! It's fixed in v6.26.1

**Comment by oallauddin** [comment]
this was issue for us. as a workaround we had to put the credentials in TF variables on the TF build server.
